src/employees/entities/employee.entity.ts

```diff
@@ -82,27 +82,21 @@ export class Employee {
   @Prop()
   prcIdNo: string;
   @Prop()
-  noticeOffense: string;
-  @Prop()
   audit201: string;
   @Prop()
-  remarks: string;
+  notes: string;
   @Prop()
   cocNo: string;
   @Prop()
-  vaccineStatus: string;
-  @Prop()
-  digitalBulletin: string;
-  @Prop()
-  viberNumber: string;
-  @Prop()
   vpdcEmail: string;
-  @Prop()
+  @Prop({ required: true })
   emergencyContactPerson: string;
-  @Prop()
+  @Prop({ required: true })
   emergencyAddress: string;
-  @Prop()
+  @Prop({ required: true })
   emergencyContactNo: string;
```

Comment:
1. Remove column **emergencyContactNo**
2. Add new column: **emergencyContact** - array of objects - [{
"name": "", "address": "", "phoneNumber": ""
 }]